.github/workflows/ci.yml

```diff
@@ -392,128 +392,131 @@ jobs:
   #               ./build_tools/scripts/check_vulkan.sh
   #               ./build_tools/cmake/ctest_all.sh ${BUILD_DIR}"
 
-  test_amd_mi250:
```

Comment:
It would be nice to keep at least one job in ci.yml running the hip HAL tests, but I want these runners to be more stable and available first... :/